hail/python/test/hail/table/test_table.py

```diff
@@ -1881,6 +1890,7 @@ def write_file(n, n_rows=5):
                     out.write('\t')
                     out.write(str(i % 2 == 0))
 
+    widths = [1 << k for k in range(8, 14)]
```

Comment:
huh, where's this from?

Reply:
this is from [dan's PR for linting the tests](https://github.com/hail-is/hail/pull/14159/files#diff-01510497ae41e1d4822af13746d443693a62febc6f3889c23aeca367aa891a65R1876), we had talked about it at standup at some point; ruff complains that widths is undefined otherwise. we can split this off into a tiny PR and just add an ignore for the error if that works better, just lmk!

Reply:
I see, that comes from #7416 and is a historical reference that is unused. Ok we can just keep it as it is in this PR.